src/libraries/Microsoft.Extensions.Logging.Abstractions/src/LogEntry.cs

```diff
@@ -0,0 +1,63 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.Extensions.Logging.Abstractions
+{
+    /// <summary>
+    /// Holds the log entry information that is being passed onto the console formatter for writing
```

Comment:
```suggestion
    /// Holds the information for a single log entry.
```

I would remove "that is being passed onto the console formatter for writing". This is only a single usage of this type.  As we add more usages of it, it doesn't make sense to call one of them out in its summary description.